setup.cfg

```diff
@@ -31,6 +31,7 @@ install_requires =
     invenio-records-resources>=5.0.0,<6.0.0
     lxml>=4.5.0
     PyYAML>=5.4.1
+    invenio-oaipmh-scythe @ git+https://github.com/ulbmuenster/invenio-oaipmh-scythe.git
```

Comment:
minor/shelve: we can move the fork under `inveniosoftware` as well once we have a fully working datastream example :)

Reply:
Sure, if we don't need a Pypi package we should.